bazaarvoice/astyanax
# PD-197980: revise backstage system information

## Jira Issue #

[PD-197980](https://bazaarvoice.atlassian.net/browse/PD-197980)

## What Are We Doing Here?

Add/revise information about components on [the Bаckstage system](https://backstage.prod.bazaarvoice.com/catalog/default/component/).

## How to Test and Verify

1. Check out this PR
2. Check if the info on [the Bаckstage system](https://backstage.prod.bazaarvoice.com/catalog/default/component/) displays correctly: astyanax

## Risk

### Level

`Low`

### Required Testing

`Smoke`

### Risk Summary

## Code Review Checklist


  we inevitably come back to this code it will only take hours to figure out, not
  days.





[PD-197980]: https://bazaarvoice.atlassian.net/browse/PD-197980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PD-197980]: https://bazaarvoice.atlassian.net/browse/PD-197980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
